template.tpl

```diff
@@ -175,11 +213,25 @@ const main = (data) => {
   // Set developer ID
   // gtagSet('developer_id.<replace_with_your_developer_id>', true);
 
-  // Set default consent state(s)
+  // Set default consent required state(s)
   let defaultData = parseConsentDefaults(data);
   defaultData.wait_for_update = 500;
+  defaultData.region = EEA_UK_COUNTRY_CODES;
+
   setDefaultConsentState(defaultData);
 
+  // Set Implicit Defaults everywhere else
```

Comment:
I probably wrote this but can we lowercase Implicit and defaults to make it consistent with the other comments?

Reply:
Yep that was from the gist but is now updated 😄